integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/synapseconfig/vfsTransport/ESBJAVA4770/axis2.xml

```diff
@@ -257,7 +257,7 @@
         <parameter name="non-blocking" locked="false">true</parameter>
```

Comment:
License needs to be updated with the year created.

Reply:
done